### ISSUE TITLE
Change config location permanently

### ISSUE THREAD

**Issue by strogiyotec** [GothenburgBitFactory/taskwarrior#2290]
Good day. I prefer to store all my dotfiles inside ~/.config folder. That is why I run task warrior with the following command.
```
task rc:~/.config/taskwarrior/.taskrc
```
And each time I have an output
```
Using alternate .taskrc file ~/.config/taskwarrior/.taskrc
[task next rc:~/.config/taskwarrior/.taskrc]
```
It's a little bit annoying. Is it a way to force taskwarrior to always look for configuration in ~/.config folder ?

**Comment by tbm** [comment]
see PR #2248 for XDG support.

**Comment by strogiyotec** [comment]
@tbm what about environment variable ? Let's say that default location would be your home directory but if you can change it by declaring variable in your .bashrc or .zshrc like 
```
export PSQL_HISTORY="$XDG_CONFIG_HOME/task/config"
```
If you are interested then I could contribute

**Comment by tunbridgep** [comment]
You could always set up an alias inside your bashrc (or whatever) file too

    alias task='task rc:~/.config/taskwarrior/.taskrc'

This is what I do for programs that don't like to follow the XDG standards but allow specifying config location

**Comment by strogiyotec** [comment]
@tunbridgep This is exactly what I am doing, however
1. I don't like an output that always points to different config location
2. It doesn't work well with zsh completion If I use alias

**Comment by XaviOndono** [comment]
[The documentation](https://taskwarrior.org/docs/configuration.html) states that this is already possible. It's not working for me though (that's how I ended up in this issue thread). Is it the docs that are too early on a non-released feature?

**Comment by Neurognostic** [comment]
> It will still show the "TASKRC OVERRIDE" message, which is kind of anoying though.
 
If you remove `header` from the list in `rc.verbose` it does not print the annoying "TASKRC OVERRIDE" message.

**Comment by tbabej** [comment]
Support for taskrc location defined by `XDG_CONFIG_HOME` implemented in #2316.
